grant_disbursement/src/main.leo

```diff
@@ -0,0 +1,235 @@
+import credits.aleo;
+import time_oracle.aleo;
+
+// The 'grant_disbursement' program.
+program grant_disbursement_puzzle.aleo {
+  // todo: update grantee rewards key
+  const GRANTEE_REWARDS_KEY: address = aleo12shtwnmf49t5atmad2jnk3e58ahtp749d9trctt9z3wryxyzt5pspp0nd0;
```

Comment:
i'd like to rename these to `GRANTEE_REWARDS_ADDRESS` and `GRANTEE_PRINCIPAL_ADDRESS`